lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -136,6 +141,7 @@ defmodule Membrane.H264.FFmpeg.Encoder do
              state.preset,
              state.profile,
              state.max_b_frames,
+             state.gop_size || -1,
```

Comment:
This is a bit inconsistent with the way `max_b_frames` is handled. Can you please unify that (I'd prefer your, simpler way)